Jenkinsfile

```
@@ -346,6 +382,7 @@ def CheckPerfDbValid(Map conf=[:]){
 ///   * "All" corresponds to "cmake -DMIOPEN_TEST_ALL=On".
 ///   * "Smoke" (-DMIOPEN_TEST_ALL=Off) is the default and usually not specified.
 ///   * "Codecov" is optional code coverage analysis.
+///   * "Performance" is optional performance analysis.
```

```suggestion
///   * "Performance Dataset" is a performance test with specified dataset.
```
Please also change line 381 to:
```
/// TestSet := { All | Smoke* | Performance Dataset } [ Codecov ]
```

---

Resolved